dev-ruby/async-io/async-io-1.33.0.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+USE_RUBY="ruby26 ruby27 ruby30 ruby31"
+RUBY_FAKEGEM_RECIPE_TEST="rspec3"
+RUBY_FAKEGEM_EXTRA_DOC="README.md"
+RUBY_FAKEGEM_GEMSPEC="${PN}.gemspec"
+inherit ruby-fakegem
+
+DESCRIPTION="Provides support for asynchonous TCP, UDP, UNIX and SSL sockets"
+HOMEPAGE="https://github.com/socketry/async-io"
+SRC_URI="https://github.com/socketry/async-io/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="$(ver_cut 1)/$(ver_cut 1-2)"
+KEYWORDS="~amd64 ~sparc"
+IUSE=""
+
+ruby_add_rdepend "dev-ruby/async"
+
+ruby_add_bdepend "test? (
+	>=dev-ruby/async-container-0.15:0
+	>=dev-ruby/async-rspec-1.10:1
+	dev-ruby/covered
```

Comment:
We try to avoid adding developer-only dependencies to packages because they bring a net negative value (there is no value for Gentoo, unlike running tests, but we still need to maintain this and this may bring blockers or additional dependency issues in the future). Please disable or patch out the use of covered and avoid the dependency.

Reply:
Done.  This actually allows me to drop the following packages:

* dev-ruby/async-http (this actually gets rid of the one test hang)
* dev-ruby/async-pool
* dev-ruby/async-rest
* dev-ruby/protocol-hpack
* dev-ruby/protocol-http1
* dev-ruby/protocol-http2
* dev-ruby/protocol-http
* dev-ruby/thread-local
* dev-ruby/traces
* dev-ruby/trenni